presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaPageSource.java

```diff
@@ -249,10 +251,21 @@ private void addColumnsRecords(QualifiedTablePrefix prefix)
         for (Map.Entry<SchemaTableName, List<ColumnMetadata>> entry : listTableColumns(session, metadata, accessControl, prefix).entrySet()) {
             SchemaTableName tableName = entry.getKey();
             int ordinalPosition = 1;
+            String columnDisplayName = "";
```

Comment:
This variable can be declared inside the look, right before `if (column.getType instanceof ...)`

Reply:
Will do.